extensions/amp-a4a/0.1/amp-ad-utils.js

```diff
@@ -162,6 +163,24 @@ export function getAmpAdMetadata(creative) {
   }
 }
 
+/**
+ * Merges any elements from customElementExtensions array into extensions array if
+ * the element is not present.
+ * @param {!Array<{custom-element: string, 'src': string}} extensions
+ * @param {!Array<string>} customElementExtensions
+ */
+export function mergeExtensionsMetadata(extensions, customElementExtensions) {
+  for (let i = 0; i < customElementExtensions.length; i++) {
+    const extensionId = customElementExtensions[i];
+    if (!extensionsHasElement(extensions, extensionId)) {
+      extensions.push({
+        'custom-element': extensionId,
+        src: `${urls.cdn}/v0/${extensionId}-0.1.js`,
```

Comment:
Let's just leave a comment here. Something like "The default version is 0.1. To specify a specific version, use `metadata['extensions']` field instead."

Reply:
done.